mon-pix/app/controllers/assessments/challenge.js

```diff
@@ -105,21 +126,35 @@ export default class ChallengeController extends Controller {
   }
 
   get displayChallenge() {
-    if (!this._isTimedChallenge) {
+    if (!this._isTimedChallenge && !this._isFocusedCertificationChallenge) {
       return true;
     }
 
     if (this._isTimedChallenge) {
       if (this.hasUserConfirmedWarning || this.model.answer || this.model.assessment.hasTimeoutChallenge) return true;
     }
 
+    if (this._hasAlreadyAnswered() || this._hasCertificationCandidateConfirmedFocusWarningScreen()) {
+      return true;
+    }
+
     return false;
   }
```

Comment:
suggestion non bloquante : j'en profiterai pour réordonner les ifs et rendre le tout plus lisible. Perso, quand c'est possible, je favorise les ifs aux ifs not.
De plus on peut simplifier l'ensemble en reorganisant légèrement. exemple :
e, déplacant la section sur les épreuves timées en premier on peut réécrire 
```js
 if (this._isTimedChallenge) {
     /*...*/
 }
if (!this._isFocusedCertificationChallenge) {
      return true;
}
```